php/commands/sidebar.php

```diff
@@ -73,6 +73,204 @@ public function list_( $args, $assoc_args ) {
 		$formatter->display_items( $wp_registered_sidebars );
 	}
 
+	/**
+	 * Reset one or all sidebars.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<sidebar-id>]
+	 * : ID for the corresponding sidebar.
+	 *
+	 * @subcommand reset
+	 */
+	public function reset( $args, $assoc_args ) {
+		if ( empty( $args ) ) {
```

Comment:
We should follow the existing pattern of requiring one or more sidebars to be passed or an `--all` flag.